app/controllers/quotes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class QuotesController < ApplicationController
+  def index
```

Comment:
See my comments in your facade. If you implement passing those two things - `:totalCount and quotes`, your controller might look like:
```
def index
  @search_phrase = params[:query]
  @total_found, @quotes = QuotesFacade.searched_quotes(@search_phrase)
end
```
Now you have access to the total count and your quotes object that is holding the attributes you want. 

The `.first(10)` could be gotten rid of if you set your limit to 10 in your service.